client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/WebDriverFactory.java

```diff
@@ -84,7 +86,9 @@ public static RemoteWebDriver createInstance(BrowserFlavors browser) {
         logger.log(Level.FINE, "Spawning a browser with the following capabilities: "
                 + DriverFactoryHelper.showCapabilities(capability));
         driver = createDriverInstance(DriverFactoryHelper.getURL(), capability);
-        DriverFactoryHelper.printDebugInfoForUser(driver);
+        if (Boolean.parseBoolean(Config.getConfigProperty(ConfigProperty.PRINT_DEBUG_INFO))) {
```

Comment:
There is a method getBoolConfigProperty in Config which would be better to use here.